x-pack/plugins/profiling/public/components/profiling_app_page_template/index.tsx

```diff
@@ -46,8 +54,34 @@ export function ProfilingAppPageTemplate({
     <ObservabilityPageTemplate
       noDataConfig={noDataConfig}
       pageHeader={{
-        pageTitle,
-        tabs,
+        rightSideItems: [
+          <EuiButton
+            href={PROFILING_FEEDBACK_LINK}
+            target="_blank"
+            color="warning"
+            iconType="editorComment"
+          >
+            {i18n.translate('xpack.profiling.header.giveFeedbackLink', {
+              defaultMessage: 'Give feedback',
+            })}
+          </EuiButton>,
+        ],
+        pageTitle: (
+          <EuiFlexGroup gutterSize="s" alignItems="baseline">
+            <EuiFlexItem grow={false}>
+              <h1>{pageTitle}</h1>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiBetaBadge
+                label="Beta"
+                color="hollow"
+                tooltipContent={i18n.translate('xpack.profiling.header.betaBadgeTooltip', {
+                  defaultMessage: 'This module is not GA. Please help us by reporting any bugs.',
```

Comment:
Do you think the message used by Uptime Synthetics (which is also in beta) would be clearer for users? Should we ask Israel if the following suggestion or another one would convey the expected level of support to the end users before GA?

```suggestion
                  defaultMessage: 'This functionality is in beta and is subject to change. The design and code is less mature than official generally available features and is being provided as-is with no warranties. Beta features are not subject to the support service level agreement of official generally available features. Please help us by reporting any bugs.',
```

Reply:
cc @iogbole

Reply:
IMO, the suggested text gives more context to the user.

Reply:
@mdbirnstiehl Could you take a look, please ?

Reply:
Is this suggestion the text that synthetics is using? If so, I think it's fine and would keep it as-is for consistency.